src/data/markdown/docs/40 xk6-disruptor/01 Get started/01 Welcome.md

```diff
@@ -5,46 +5,76 @@ head_title: 'xk6-disruptor Documentation'
+Key features include:
 
-The extension offers an [API](/javascript-api/xk6-disruptor/api/) to create disruptors that target one specific type of component (for example, Pods).
-These disruptors can inject different types of [faults](/javascript-api/xk6-disruptor/api/faults), such as errors in HTTP requests, served by that component.
-Currently, disruptors exist for [Pods](/javascript-api/xk6-disruptor/api/poddisruptor) and [Services](/javascript-api/xk6-disruptor/api/servicedisruptor), but others will be introduced in the future, along with other types of faults for existing disruptors.
+- Everything as code. Facilitate test reuse and collaboration between teams without learning a new DSL.
 
-<Blockquote mod="note">
+- Fast to adopt with no day-two surprises. [No need to deploy and maintain](/javascript-api/xk6-disruptor/explanations/how-xk6-disruptor-works/) a fleet of agents or operators.
 
-xk6-disruptor is intended for systems running in Kubernetes. Other platforms are not supported at this time.
+- Easy to extend and integrate with other types of k6 tests. No need to try to glue multiple tools together to get the job done.
 
-</Blockquote>
+## Capabilities
 
-## Use case for xk6-disruptor
+Currently, the disruptor is intended to test applications running in Kubernetes. Other platforms are not supported at this time.
 
-The main use case for xk6-disruptor is to test how resilient an application is to diverse types of disruptions. xk6-disruptor can reproduce the effects of these disruptions, without having to reproduce their root causes.
-For example, you can inject delays in the HTTP requests that an application makes to a service  without having to stress or interfere with the infrastructure (network, nodes) on which the service runs.
+It provides a Javascript API to inject different [faults](/javascript-api/xk6-disruptor/api/faults/) in HTTP and gRPC requests, such as errors and delays, into the selected Kubernetes [Pods](javascript-api/xk6-disruptor/api/poddisruptor) or [Services](/javascript-api/xk6-disruptor/api/servicedisruptor).
 
-In this way, xk6-disruptor makes reliability tests repeatable, predictable, and with a limited blast radius.
-These characteristics make it easier to incorporate such tests into the test suites of applications that deploy on shared infrastructures (such as staging environments).
+Other types of faults and disruptors will be introduced in the future. The [Roadmap](https://github.com/grafana/xk6-disruptor/blob/main/ROADMAP.md) presents the project's goals for the coming months regarding new functionalities and enhancements.
 
+```javascript
+export default function () {
+    // Create a new pod disruptor with a selector
+    // that matches pods from the "default" namespace with the label "app=my-app"
+    const disruptor = new PodDisruptor({
+        namespace: "default",
+        select: { labels: { app: "my-app" } },
```

Comment:
Dumb suggestion for recommended best practices https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/
```suggestion
        select: { labels: { "app.kubernetes.io/name": "my-app" } },
```